plugins/org.jkiss.dbeaver.ui.editors.sql/src/org/jkiss/dbeaver/ui/controls/ScriptSelectorPanel.java

```diff
@@ -190,7 +190,15 @@ public void widgetSelected(SelectionEvent e) {
         } else {
             UIUtils.createPlaceholder(childComposite, 1);
         }
-        Button projectCheckbox = UIUtils.createCheckbox(childComposite, SQLEditorMessages.script_selector_project_scripts, false);
+        Button projCheckbox = UIUtils.createCheckbox(childComposite, SQLEditorMessages.script_selector_project_scripts, false);
```

Comment:
Why this change?

Reply:
Checkstyle was worried about 12 lines between declaration and usage of this checkbox, so I moved it up, but the moved line is too long, so I decided to rename it.

Reply:
It seems to be only 130 characters while the limit is 140. So you can keep its original name.

Reply:
I have already fixed it by introducing the local variable for preference store. Line 199 was more than 140 characters.

Reply:
We are talking about line 193

Reply:
Please, see latest changes. It's outdated